zkevm-circuits/src/evm_circuit/execution/error_oog_exp.rs

```diff
@@ -204,24 +205,17 @@ mod tests {
         let code_b = testing_data.bytecode.clone();
         let gas_cost_b = testing_data.gas_cost;
 
-        // Code A calls code B.
-        let code_a = bytecode! {
-            // populate memory in A's context.
-            PUSH8(U256::from_big_endian(&rand_bytes(8)))
-            PUSH1(0x00) // offset
-            MSTORE
-            // call ADDR_B.
-            PUSH1(0x00) // retLength
-            PUSH1(0x00) // retOffset
-            PUSH32(0x00) // argsLength
-            PUSH32(0x20) // argsOffset
-            PUSH1(0x00) // value
-            PUSH32(addr_b.to_word()) // addr
-            // Decrease expected gas cost (by 1) to trigger out of gas error.
-            PUSH32(gas_cost_b - 1) // gas
-            CALL
-            STOP
-        };
+        // code A calls code B.
+        let pushdata = rand_bytes(32);
```

Comment:
Only used in `MockBytecodeParams` so no need to store it here in a new variable, can just set it directly in `MockBytecodeParams`? Same with `gas_cost_b`?

Reply:
Updated. Set `pushdata` and  `gas_cost_b` directly.